u6-spas/src/views/EditProfile.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div class="profile-edit">
+    <div>
+      <h1>{{ name }}</h1>
+      {{ tagline }}
+      <ul>
+        <img :src="picUrl" width="300" />
+      </ul>
+    </div>
+    <div>
+      <input type="text" v-model.number="name" />
+      <input type="text" v-model.number="tagline" />
+      <input type="text" v-model.number="picUrl" />
```

Comment:
คุณสมบัติเหล่านี้เป็น `string`s ดังนั้นคุณไม่ควรใช้ `v-model.number`; ใช้เฉพาะ `v-model` ที่นี่
These properties are `string`s, so you should not use `v-model.number`; only use `v-model` here.